edge/pkg/common/util/util.go

```diff
@@ -0,0 +1,75 @@
+package util
+
+import (
+	"bytes"
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/json"
+	"fmt"
+	"net"
+	"net/http"
+	"os"
+	"time"
+
+	"k8s.io/klog/v2"
+
+	"github.com/kubeedge/kubeedge/pkg/apis/componentconfig/edgecore/v1alpha2"
+	"github.com/kubeedge/kubeedge/pkg/apis/operations/v1alpha1"
+	"github.com/kubeedge/kubeedge/pkg/util/fsm"
+)
+
+const ISO8601UTC = "2006-01-02T15:04:05Z"
```

Comment:
`ISO8601UTC` is defined in two places, Could you extract it to `pkg/util`?

Reply:
This is due to historical considerations. The outer `pkg` contains references to the `edge`. This results in circular dependencies. We can create an issue to address this issue.